graph/src/components/store.rs

```diff
@@ -41,6 +42,19 @@ pub struct EntityKey {
 
     /// ID of the individual entity.
     pub entity_id: String,
+    // Note: Do not modify fields without making a backward compatible change to
+    // the StableHash impl (below)
```

Comment:
It would be worth describing what a backward-compatible change would look like? Or is this covered in the stable_hash crate docs already?

Reply:
The RFC covers backward compatibility but should be moved into the crate docs when time allows.